app/views/income_details_per_member/_household_member_group.html.erb

```diff
@@ -0,0 +1,6 @@
+<fieldset class="form-group household-member-group" data-member-name="<%= member.display_name %>">
+  <legend class="text--section-header">
+    <%= member.display_name %> (<%= member.employment_status.titleize %>)
+  </legend>
+  <%= yield %>
+</fieldset>
```

Comment:
This is a layout that can be used to wrap a set of household member fields in a fieldset